### ISSUE TITLE
Deduplicate deprecation warnings

### ISSUE THREAD

**Issue by RalfJung** [coq/coq#10148]
#### Description of the problem

When an ltac function used throughout all proof scripts gets deprecated, the build log just drowns in 1000s of times of the same message being repeated. You can see that, for example, [here](https://gitlab.mpi-sws.org/iris/lambda-rust/-/jobs/31331).

It would be great if the warning could be deduplicated to be printed only once per `coqc` invocation. That would keep the build log readable.

(FWIW, we cannot do anything about this warning currently other than silence it, because `change_no_check` does not exist in most Coq versions we support.)

#### Coq Version

master

**Comment by SkySkimmer** [comment]
>(FWIW, we cannot do anything about this warning currently other than silence it, because change_no_check does not exist in most Coq versions we support.)

The warning is specific to change_no_check, so what's wrong with disabling it until you're ready to change supported Coq versions?

**Comment by RalfJung** [comment]
Nothing, I guess -- except that the warning thus uses all utility for us. Which is okay, but might still be useful feedback for you guys.

**Comment by Zimmi48** [comment]
No, disabled warnings can still be useful (as long as you don't disable all categories at once) if you make all other warnings fatal as they indicate what will need to be fixed in the future. Example here: https://github.com/coq-community/lemma-overloading/blob/master/_CoqProject

**Comment by ejgallego** [comment]
I'm tempted to solve this a wontfix , because I don't see a strategy to solve this bug.

**Comment by SkySkimmer** [comment]
Agree wontfix
